index.d.ts

```diff
@@ -196,7 +196,11 @@ export declare class Image {
     options?: { border?: Array<number> },
   ): Array<number>;
 
-  // cannyEdge
+  cannyEdge(options?: {
+    gaussianBlur: number;
+    lowThreshold: number;
+    highThreshold: number;
```

Comment:
Those are all optional, the default value will be used for the missing properties.